### PR TITLE
Update 'Director, Ecosystem' job description

### PR DESCRIPTION
**Pull request by WrathfulSpatula**
This updates the copy for the "Director, Ecosystem" job description. It also updates a `yaml` dependency version. (I need to see the preview environment, to know if the latter change will work.)

### CODE REVIEW

**Comment by WrathfulSpatula** [comment]
@willzeng I gave a very thorough attempt at updating all `yaml` versions: I think I can say it is literally not _possible_ to do so, right now. Firstly, `pnpm update` breaks the app deployment, (probably due to version incompatibilities with our code). Secondly, these are second-order dependencies on packages that, themselves, have not updated their `yaml` version.

There's no way to do this, until dependencies update their second-order dependencies. Then, when we update those dependencies, it will likely be necessary to modify our app's code. (There's just no way, for now.)

File: src/content/job/director_ecosystem.md [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-This position is full time with a competitive salary range, commensurate with experience. Unitary Fund is a fully remote team and this role is also remote.
+## Role - Director, Ecosystem
+You will build sustainable and community-oriented open source practices to ensure the tools of the quantum ecosystem reach more people, sooner. This includes development and execution of in-person and online events, as well as outreach strategies to build new audiences and ensure support for existing users. You will work with a diverse array of community members including Unitary Fund Donors, Ambassadors, Advisory Board Members, Grantees, users advocates and more.
+
+**This position is full time with a competitive salary range, commensurate with experience. Unitary Fund is a fully remote team and this role is also remote.**
 
 ## About You:
-- 3+ years in community engagement/management experience with diverse set of stakeholders with an emphasis on growth
+- 4-6 years in community engagement/management experience with diverse set of stakeholders with an emphasis on growth
```
Comment (willzeng) [inline src/content/job/director_ecosystem.md:22]:
```suggestion
- 4-6 years in ecosystem engagement/management experience with diverse set of stakeholders with an emphasis on growth
```

**Review by willzeng — APPROVED** [review]

**Comment by nathanshammah** [comment]
The bot renders well, so I am going to merge.

File: src/content/job/director_ecosystem.md [hunk 2]
```diff
@@ -1,31 +1,33 @@
 ---
-title: Community Manager, Full Time, Remote
+title: Director, Ecosystem
 ---
-Posted on August 18th, 2023. 
+Posted on August 18th, 2023.
```
Comment (WrathfulSpatula) [inline src/content/job/director_ecosystem.md:4]: @beecastanon Do you mean, remove this?  Yes, we can do that. (Please just clarify that this is the "timestamp" to which you refer.)

Comment (beecastanon) [reply src/content/job/director_ecosystem.md:4]: @WrathfulSpatula thanks! "Posted on August 18th, 2023. Updated on September 19th, 2023." is the text that I would imagine may be confusing to folks.

**Pull request merged**